sanic/server/socket.py

```diff
@@ -112,6 +112,17 @@ def configure_socket(
                 server_settings["port"],
                 backlog=backlog,
             )
+        except PermissionError:
+            p = server_settings["port"]
+            if not p or p >= 1024:
+                raise
+            addr = f"{server_settings['host']}:{p}"
+            error = ServerError(
+                f"Permission denied binding to {addr}.\n\n"
+                "Use `sudo sanic` to run on a privileged port.\n"
```

Comment:
Rather than assuming the command was `sanic` we can pull from `sys` to try and give a more friendly reproduction of what they used.

Reply:
Seems a bit overkill but I can have a look if there are no corner cases that need handling other than prepending `sudo` to produce the full command (I recall Python does some magic with argv so it isn't really what the system sent).

One possible concern is also with the OSError handler below, which I presume is meant for EADDRINUSE (98), but with diagnostic for the WorkerManager FAQ. We can also see various other errors not really handled here, in particular with unix sockets. I could add a few more handlers for the common problems with better diagnostics, and some tests.